Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```diff
@@ -0,0 +1,1171 @@
+category: Network Security
+commonfields:
+  id: AzureKeyVault
+  version: -1
+configuration:
+- display: Client ID
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret (only when using self-deployed Azure application)
+  name: client_secret
+  required: false
+  type: 4
+- display: Use a self-deployed Azure application
+  hidden: false
+  name: self_deployed
+  required: false
+  type: 8
+- display: Tenant ID
+  name: tenant_id
+  required: true
+  type: 0
+- display: Subscription ID
+  name: subscription_id
+  required: true
+  type: 0
+- display: Resource Group Name
+  name: resource_group_name
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  hidden: false
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
```

Comment:
let's make sure these two are the last parameters in the configuration

Reply:
Done